common/service/dynamicconfig/constants.go

```diff
@@ -827,6 +830,12 @@ const (
 	WorkerThrottledLogRPS
 	// ScannerPersistenceMaxQPS is the maximum rate of persistence calls from worker.Scanner
 	ScannerPersistenceMaxQPS
+	// ScannerGetOrphanTasksPageSize is the maximum number of orphans to delete in one batch
+	ScannerGetOrphanTasksPageSize
+	// ScannerBatchSizeForCompleteTasksLessThanAckLevel is the number of tasks to query per call in the scavenger
+	ScannerBatchSizeForCompleteTasksLessThanAckLevel
+	// ScannerMaxTasksProcessedPerTasklistJob is the number of tasks to process before letting another task list be processed
```

Comment:
Also need to fix the comment/description for ScannerMaxTasksProcessedPerTasklistJob and ScannerBatchSizeForCompleteTasksLessThanAckLevel

maybe:
```
ScannerMaxTasksProcessedPerTasklistJob is the number of tasks to process for a tasklist in each workflow run
```
and
```
ScannerBatchSizeForCompleteTasksLessThanAckLevel is the batch size passed into CompleteTasksLessThanAckLevel to delete completed tasks. When the number of deleted tasks is less than the batch size, the tasklist will also be deleted. 
```